com.pypyrev.linuxdcpp.json

```diff
@@ -0,0 +1,74 @@
+{
+	"app-id": "com.pypyrev.linuxdcpp",
+	"runtime": "org.freedesktop.Platform",
+	"runtime-version": "20.08",
+	"sdk": "org.freedesktop.Sdk",
+	"command": "linuxdcpp",
+	"finish-args": [
+		"--socket=x11",
+		"--share=ipc",
+		"--share=network",
+		"--filesystem=host"
+	],
+	"rename-desktop-file": "linuxdcpp.desktop",
+	"rename-icon": "linuxdcpp",
+	"modules": [
+		"shared-modules/gtk2/gtk2.json",
+		{
+			"name": "boost",
+			"sources": [{
+					"type": "archive",
+					"url": "https://boostorg.jfrog.io/artifactory/main/release/1.76.0/source/boost_1_76_0.tar.bz2",
+					"sha256": "f0397ba6e982c4450f27bf32a2a83292aba035b827a5623a14636ea583318c41"
+				}],
+			"buildsystem": "simple",
+			"build-commands": [
+				"mv boost $CPLUS_INCLUDE_PATH"
+			],
+			"cleanup": ["/include"]
+		},
+		{
+			"name": "libnotify",
+			"sources": [{
+					"type": "archive",
+					"url": "https://github.com/GNOME/libnotify/archive/refs/tags/0.7.9.tar.gz",
+					"sha256": "9bd4f5fa911d27567e7cc2d2d09d69356c16703c4e8d22c0b49a5c45651f3af0"
+				}],
+			"buildsystem": "meson",
+			"build-options": {
+				"config-opts": ["-Dman=false"]
+			},
+			"cleanup": ["/include"]
+		},
+		{
+			"name": "libglade2",
+			"sources": [{
+					"type": "archive",
+					"url": "https://download.gnome.org/sources/libglade/2.6/libglade-2.6.4.tar.bz2",
+					"sha256": "64361e7647839d36ed8336d992fd210d3e8139882269bed47dc4674980165dec"
+				}],
+			"buildsystem": "autotools",
+			"cleanup": ["/include"]
+		},
+		{
+			"name": "linuxdcpp",
+			"sources": [{
+					"type": "archive",
+					"url": "http://pypyrev.com/download/linuxdcpp-1.1.0.1.tgz",
+					"sha256": "839365cf30d91cf196f0a3dcf5425101f2c6c93f3758ce3a1365f1c2efb3312f"
+				}, {
+					"type": "file",
+					"path": "com.pypyrev.linuxdcpp.appdata.xml"
```

Comment:
has this been submitted upstream?

Reply:
For now I don't see a need of appdata.xml in upstream. So, let it be a separate file specific to flathub. Of course, I'll add it to upstream if it will be needed anywhere else.

Reply:
appstream files are NOT specific to flathub. so yes having it upstream is useful.